tests/ballerina-integration-test/src/test/java/org/ballerinalang/test/service/http/sample/HttpPipeliningTestCase.java

```diff
@@ -94,8 +97,10 @@ public void testPipeliningWithTimeout() throws IOException, InterruptedException
     public void testPipeliningLimit() throws IOException, InterruptedException {
         HttpClient httpClient = new HttpClient(HOST, 9222);
         String connectionCloseMsg = httpClient.sendMultiplePipelinedRequests("/pipeliningLimit/testMaxRequestLimit");
-        assertEquals(connectionCloseMsg, "Channel is inactive");
-
+        assertTrue(CHANNEL_INACTIVE.equals(connectionCloseMsg) ||
+                CONNECTION_RESET.equals(connectionCloseMsg), "When the channel is closed from the server" +
+                "side, client will either receive a channel inactive message or if another pipeline request is" +
+                "being written to the closed connection, a connection reset message");
```

Comment:
Small suggestion, shall we say "client should" instead of "client will" since this is a message for when assertion fails? Also, shall we move the operators to the new lines and add spaces where required at the sentence breaks?